programs/x509/cert_write.c

```diff
@@ -466,7 +468,8 @@ int main( int argc, char *argv[] )
     if( ( ret = mbedtls_mpi_read_string( &serial, 10, opt.serial ) ) != 0 )
     {
         mbedtls_strerror( ret, buf, 1024 );
-        mbedtls_printf( " failed\n  !  mbedtls_mpi_read_string returned -0x%02x - %s\n\n", -ret, buf );
+        mbedtls_printf( " failed\n  !  mbedtls_mpi_read_string "
+                        "returned -0x%02x - %s\n\n", -ret, buf );
```

Comment:
I think that some of these functions actually encode the error code in two bytes, so the printing might be a bit off

Reply:
As far as I see, functions from the bignum module only return low-level module codes, but as such code is likely to be copy-pasted to other contexts where longer error codes can be returned, it's probably safer to unconditionally use two byte output.